modules/hcp-bm-hosts.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-disconnected/hcp-deploy-dc-bm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-bm-hosts_{context}"]
+= Creating bare metal hosts for the hosted cluster
+
+A _bare metal host_ is an `openshift-machine-api` object that encompasses physical and logical details so that it can be identified by a Metal3 Operator. Those details are associated with other Assisted Service objects, known as _agents_.
+
+.Prerequisites
+
+Before you create the bare metal host and destination nodes, you must have the destination machines ready.
+
+.Procedure
+
+To create a bare metal host, complete the following steps:
+
+. Create a YAML file with the following information:
++
+Because you have at least one secret that holds the bare metal host credentials, you need to create at least two objects for each worker node.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <hosted_cluster_name>-worker0-bmc-secret <1>
```

Comment:
```suggestion
  name: <hosted_cluster_name>-worker0-bmc-secret \// <1>
```
Suggestion to add comments for all the call outs in this YAML.

Reply:
Will do